src/oc_erchef/apps/chef_index/src/chef_elasticsearch.erl

```diff
@@ -45,33 +45,11 @@ search(#chef_solr_query{} = Query) ->
     end.
 
 handle_successful_search(ResponseBody) ->
-    EjsonBody = jiffy:decode(ResponseBody),
-    ScrollId = ej:get({<<"_scroll_id">>}, EjsonBody),
-    Response = ej:get({<<"hits">>}, EjsonBody),
+    Response = ej:get({<<"hits">>}, jiffy:decode(ResponseBody)),
     NumFound = ej:get({<<"total">>}, Response),
     DocList  = ej:get({<<"hits">>}, Response),
     Ids = [ ej:get({<<"_id">>}, Doc) || Doc <- DocList ],
-    scroll(ScrollId, NumFound, length(Ids), Ids).
-
-scroll(ScrollId, NumFound, NumFound, Ids) ->
-    ok = chef_index_http:delete("/_search/scroll", ScrollId),
-    {ok, undefined, NumFound, Ids};
-scroll(ScrollId, NumFound, _, Ids) ->
-    Url = "/_search/scroll?scroll=1m",
-    {ok, Code, _Head, Body} = chef_index_http:request(Url, get, ScrollId),
-    case Code of
-        "200" ->
-            DocList = ej:get({<<"hits">>, <<"hits">>}, jiffy:decode(Body)),
-            NewIds = [ ej:get({<<"_id">>}, Doc) || Doc <- DocList ],
-            AllIds = lists:append([ Ids, NewIds ]),
-            scroll(ScrollId, NumFound, length(AllIds), AllIds);
-        %% For now keep these error messages
-        %% consistent with chef_solr
-        "400" ->
-            {error, {solr_400, Url}};
-        "500" ->
-            {error, {solr_500, Url}}
-    end.
+    {ok, undefined, NumFound, Ids}.
```

Comment:
I think this was an existing problem, but since we return undefined here, it seems like our spec for search() above is a bit wrong.